addon/helpers/d3-tick-format.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+import { timeFormat } from 'd3-time-format';
+
+export function d3TickFormat([tickFormat]) {
```

Comment:
That's an annoying problem, what about doing something like this https://github.com/ivanvanderbyl/maximum-plaid/blob/master/addon/helpers/format-fn.js